src/pheval/runners/runner.py

```diff
@@ -14,6 +22,91 @@ class PhEvalRunner(ABC):
     output_dir: Path
     config_file: Path
     version: str
+    directory_path = None
+
+    def _get_tool(self):
+        return parse_input_dir_config(self.input_dir).tool
+
+    def _get_phenotype_only(self):
+        return parse_input_dir_config(self.input_dir).phenotype_only
```

Comment:
From the parse_input_dir_config method, it seems the parameter it expects the path to a config file, not a directory.

Reply:
Also, I think its ok to read this file over and over again, but maybe just reading it once would be a better design. Up to you.

Reply:
I will alter to just reading once

Reply:
added a __post_init__ method that reads the input config once